ui/css/treeherder-global.css

```diff
@@ -22,25 +22,13 @@ a:visited {
   color: purple;
 }
 
-input:focus::-webkit-input-placeholder {
-  color: transparent;
-}
-
-input:focus::-moz-placeholder {
-  color: transparent;
-}
-
```

Comment:
These `placeholder`s aren't on a standards track, and don't appear to be used in any inputs I could find.
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-input-placeholder